.github/workflows/translations.yaml

```diff
@@ -0,0 +1,65 @@
+name: Translations
+
+on:
+  schedule:
+    - cron: "30 0 * * *"
+  push:
+    branches:
+      - dev
+    paths:
+      - translations/en.json
+
+env:
+  NODE_VERSION: 12
+
+jobs:
+  upload:
+    name: Upload
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout the repository
+        uses: actions/checkout@v2
+
+      - name: Set up Node ${{ env.NODE_VERSION }}
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{ env.NODE_VERSION }}
+
+      - name: Upload Translations
+        run: |
+          export LOKALISE_TOKEN="${{ secrets.LOKALISE_TOKEN }}"
+
+          ./script/translations_upload_base
+
+  download:
+    name: Download
+    needs: upload
+    if: github.event_name == 'schedule'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout the repository
+        uses: actions/checkout@v2
+
+      - name: Set up Node ${{ env.NODE_VERSION }}
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{ env.NODE_VERSION }}
+
+      - name: Download Translations
+        run: |
+          export LOKALISE_TOKEN="${{ secrets.LOKALISE_TOKEN }}"
+
+          npm install
+          ./script/translations_download
+
+      - name: Initialize git
+        uses: home-assistant/actions/helpers/git-init@master
+        with:
+          name: GitHub Action
+          email: github-action@users.noreply.github.com
+
+      - name: Update translation
+        run: |
+          git add translation
+          git commit -am "[ci skip] Translation update"
```

Comment:
`[ci skip]` doesn't work with GitHub Actions.

Should the author be added here? `--author="$GITHUB_ACTOR <$GITHUB_ACTOR@users.noreply.github.com>"` ?

Reply:
That I only kept being consistent with what we had.
This will be committed as github action, and that will not start any workflows anyway